examples/tls-inspector/verify.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+export NAME=tls-inspector
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+run_log "Test tls inspector"
+responds_without \
+    '"x-forwarded-proto"' \
+    -k \
+    https://localhost:10000
+curl -sk --http1.1 https://localhost:10000  | jq  '.os.hostname'
```

Comment:
maybe add 

```
| grep service-https-http1.1
```
to these so they are testing the response

Reply:
Got it.